backend/src/langchain.ts

```diff
@@ -195,17 +151,15 @@ async function queryPromptEvaluationModel(
 
 function formatEvaluationOutput(response: string) {
 	// remove all non-alphanumeric characters
-	try {
-		const cleanResponse = response.replace(/\W/g, '').toLowerCase();
+	const cleanResponse = response.replace(/\W/g, '').toLowerCase();
+	if (cleanResponse === 'yes' || cleanResponse === 'no') {
```

Comment:
Replaced this bit of logic because I'm pretty certain the original try/catch was doing nothing for us.

Reply:
I'd be interested to know what the response actually is here, and why we need to strip out all non-alphanumerics; does it wrap the "yes" or "no" inside braces, or quotes maybe?